beluga-tunnel/src/proto.rs

```diff
@@ -16,7 +16,9 @@ pub(crate) struct Message {
     pub connection_id: u32,
 }
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, prost::Enumeration, enumn::N)]
+#[derive(
+    Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, prost::Enumeration, enumn::N,
```

Comment:
Do we still need `enumn::N` here? `prost::Enumeration` already provides a `TryFrom<i32>` implementation.